### DISCUSSION TITLE
Ability to get chat/group history

### DISCUSSION THREAD

**Discussion by alexzhyshko** [rubenlagus/TelegramBots#1448] [Q&A]
Hi all.

Using this library for some time already and just can't find a function to get chat/group history in it.
Checked documentation thinking it is not possible, however considering this link: https://core.telegram.org/method/messages.getHistory or this one https://core.telegram.org/method/channels.getMessages it should be possible.

Maybe not finding what is already there or this wasn't implemented yet / not supported?

Please, help me to figure this out )

**Comment by Chase22** [comment]
The HTTP Api this library is based on does not allow access to the group history. The method you've found is from mtproto, a different API to access telegram. 

So it's not an issue with the library
